apps/hash-frontend/src/pages/types/types-page-tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import { Tabs, tabsClasses, typographyClasses } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { TabLink } from "../../shared/ui/tab-link";
+import { TabId } from "./[[...type-kind]].page";
+
+const tabIds = [
+  "all",
+  "entity-type",
+  "link-type",
+  "property-type",
+  "data-type",
+] satisfies TabId[];
+
+export const tabTitles: Record<TabId, string> = {
+  all: "All",
+  "entity-type": "Entity Types",
+  "link-type": "Link Types",
+  "property-type": "Property Types",
+  "data-type": "Data Types",
+};
+
+type TypesPageTabsProps = {
+  currentTab: TabId;
+  numberOfTypesByTab: Record<TabId, number | undefined>;
+};
+
+export const TypesPageTabs: FunctionComponent<TypesPageTabsProps> = ({
+  currentTab,
+  numberOfTypesByTab,
+}) => {
+  return (
+    <Tabs
+      value={currentTab}
+      TabIndicatorProps={{
+        sx: ({ palette }) => ({
+          height: 3,
+          backgroundColor: palette.blue[60],
+          minHeight: 0,
+          bottom: -1,
+        }),
+      }}
+      sx={{
+        minHeight: 0,
+        overflow: "visible",
+        alignItems: "flex-end",
+        flex: 1,
+        [`.${tabsClasses.scroller}`]: {
+          overflow: "visible !important",
+        },
+      }}
+    >
```

Comment:
This looks identical to the equivalent component in `entity-type-tabs.tsx`, can we share it also? e.g. some kind of `TabsContainer`. Ideally all these tabs would look identical and only be defined in one place

Reply:
Yeah thanks for reminding me to make this a re-usable component, have done so in https://github.com/hashintel/hash/pull/2908/commits/f732973c3c3369daaaa65a5b8bf14976d79e34ce and also moved the style overrides for `Tabs` and `Tab` into the theme rather than defining them inline.